content/shmem_g.tex

```diff
@@ -23,11 +23,14 @@
   \apiargument{IN}{source}{Symmetric address of the source data object.
     The type of \source{} should match that implied in the SYNOPSIS section.}
   \apiargument{IN}{pe}{The number of the remote \ac{PE} on which \VAR{source} resides.}
+  \apiargument{IN}{pe}{\ac{PE} number of the remote \ac{PE} on which \VAR{source} resides 
+    relative to the team associated with the given \VAR{ctx} when provided, or the 
+    default context otherwise.}
```

Comment:
```suggestion
  \apiargument{IN}{pe}{\ac{PE} number of the remote \ac{PE} on which \VAR{source} resides 
    relative to the team associated with the given \VAR{ctx} when provided, or the 
    default context otherwise.}
```